Jenkinsfile-Agent-Test-3

```diff
@@ -75,6 +76,11 @@ pipeline {
                                 sh 'bash test/plugin/run.sh --build_id=wl3_${BUILD_ID} canal-scenario'
                             }
                         }
+                        state('sofarpc 5.4.0-5.6.2 (23)') {
+                            steps {
+                                sh 'bash test/plugin/run.sh --build_id=${BUILD_ID} sofarpc-scenario'
```

Comment:
`--build_id=wl3_${BUILD_ID}` should replace --build_id=${BUILD_ID}

Reply:
done